test/test_images/httpproxy/service.yaml

```diff
@@ -11,3 +11,11 @@ spec:
         env:
         - name: TARGET_HOST
           value: "helloworld-test-image.default.svc.cluster.local"
+        - name: PROBE_PORT
+          value: 9090
+        startupProbe:
```

Comment:
AFAIK we haven't plumbed startupProbe through, so not sure how this could work.

Reply:
I also didn't think this was available until like 1.18

Reply:
I haven't tested it yet, I was relying on the E2E tests to stop me :smile: 
You are correct, it is not allowed: https://github.com/knative/serving/blob/master/pkg/apis/serving/fieldmask.go#L227
How does this pass validation?

Reply:
The yamls are just samples for folks, and we use them to publish the image.  You'll also have to fix whatever code is constructing services with this image.

Reply:
There is no validation: https://github.com/knative/serving/blob/591510c0c917d4b45ea919568181a75426b543e0/pkg/apis/serving/k8s_validation.go#L374

Reply:
It is just dropped.

Reply:
This should do it: https://github.com/knative/serving/blob/591510c0c917d4b45ea919568181a75426b543e0/pkg/apis/serving/k8s_validation.go#L400

Reply:
Yes, I am just very dumb.
The YAML under `test/test_images/*` are simply used to upload the images with `ko resolve` :expressionless: